config/metadata/dependencies.yaml

```diff
@@ -6,4 +6,4 @@ dependencies:
 - type: olm.package
   value:
     packageName: ocs-operator
-    version: ">4.7.0 <4.8.0"
+    version: ">=4.7.0 <4.8.0"
```

Comment:
The catalogsource that deploy-with-olm.sh deploys only has ocs-operator 4.7.0. I added this version to dependencies.yaml so that the operator will install with this catalogsource.

Reply:
As of now, this is ok, but we will need to decouple the catalog source from our deploy script as it currently hard coded against my private repo